CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 4.1.1 (Unreleased)
```

Comment:
If there are enhancements than this should be a `4.2.0` release to follow semver.

Reply:
```suggestion
## 4.2.0 (Unreleased)
```